client/app/scripts/liveblog-edit/views/post.html

```diff
@@ -69,7 +69,7 @@
                 </div>
                 <div ng-if="!post.multiple_items" class="lb-post__list">
                     <div ng-repeat="item in post.items track by item.item._id">
-                        <div lb-bind-html html-content="{{ item.item.text }}" class="lb-post__item" ng-class-even="'lb-post__item--even'"></div>
+                        <div lb-bind-html html-content="{{ item.item.text | fix_http_protocol }}" class="lb-post__item" ng-class-even="'lb-post__item--even'"></div>
```

Comment:
Can we apply this expensive filter only for uploaded picture ?
Because we maybe don't want to convert all the links in items. For external link, `https` may not be available.